libc/src/string/memmove.c

```diff
@@ -0,0 +1,10 @@
+#include <string.h>
+
+void *memmove(void *dest, const void *src, size_t n){
+    const char *from = (const char*)src;
+    char *to = (char*)dest;
+    char data[n];
```

Comment:
You are creating a variable size array here. This will fail if n is too large (we basically run out of stack). Try to implement this without any additional allocation.